app/src/main/java/org/p2p/wallet/send/ui/main/SendFragment.kt

```diff
@@ -124,16 +122,22 @@ class SendFragment :
                     val ordinal = result.getInt(EXTRA_NETWORK, 0)
                     presenter.setNetworkDestination(NetworkType.values()[ordinal])
                 }
-                result.containsKey(EXTRA_RESULT_KEY_DISMISS) -> {
-                    popBackStack()
-                }
             }
         }
 
+        // childFragmentManager for BottomSheets
+        childFragmentManager.setFragmentResultListener(KEY_REQUEST_SEND, viewLifecycleOwner) { _, result ->
+            if (result.containsKey(EXTRA_RESULT_KEY_DISMISS)) closeScreenAndReturn()
+        }
+
         presenter.loadInitialData()
         checkClipBoard()
     }
 
+    private fun closeScreenAndReturn() {
+        if (backStackEntryCount() > 1) popBackStack() else replaceFragment(HomeFragment.create())
```

Comment:
is it ok to create HomeFragment here from scratch? maybe i can use bottom tabs to switch to Home??

Reply:
I think we already have an ability to switch between tabs if you call `popBackStack()`

Reply:
popBackStack closes the app itself. I think we are replacing fragments when switching tabs

Reply:
Look into `MainFragment`, there is already implemented switching between tabs when onBackPressed
```
override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requireActivity().onBackPressedDispatcher.addCallback(this) {
            if (binding.bottomNavigation.selectedItemId != R.id.itemHome) {
                navigate(R.id.itemHome)
                binding.bottomNavigation.menu[0].isChecked = true
            } else {
                requireActivity().finish()
            }
        }
    }
```

Reply:
+++